assets/css/main.css

```diff
@@ -82,15 +82,20 @@ ul li {
 }
 
 .btn-accent{
-  background: #22b573;
+  background: white;
   -webkit-box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
   -moz-box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-  color: white;
+  color: #16754A;
+  transition: all .2s linear;
 }
 
 .btn-accent:hover {
-    background: #a2ea6a;
+    background: #E0E0E0 !important;
```

Comment:
I think the hover color we had earlier made it easier to identify that the button was clickable (the hover color had more contrast)
```
background: #a2ea6a;
color: #000000;
```